### PR TITLE
Create default constraint endpoint

### LINKED ISSUE #7255: OPA Default Constraint - create endpoint
**User Story**
Implement OPA Default Constraint - create endpoint

### PR DESCRIPTION
**Pull request by imharshita**
Signed-off-by: Harshita <harshita.sharma6174@gmail.com>

**What this PR does / why we need it**:
Adds the create default constraint endpoint

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #7255

**Does this PR introduce a user-facing change?**:

```release-note
new endpoint for default constraint create: POST /api/v2/constraints
```

### CODE REVIEW

**Review by irozzo-1A — COMMENTED** [review]
/approve

File: pkg/handler/v2/constraint/constraint.go [hunk 1]
```diff
@@ -531,3 +531,41 @@ func DecodePatchConstraintReq(c context.Context, r *http.Request) (interface{},
...
+		adminUserInfo, err := userInfoGetter(ctx, "")
+		if err != nil {
+			return nil, err
+		}
+		if !adminUserInfo.IsAdmin {
+			return nil, utilerrors.New(http.StatusForbidden,
+				fmt.Sprintf("forbidden: \"%s\" doesn't have admin rights", adminUserInfo.Email))
+		}
+
+		constraint := convertAPIToInternalConstraint(req.Body.Name, ConstraintNamespace, req.Body.Spec)
```
Comment (lsviben) [inline pkg/handler/v2/constraint/constraint.go]: This ConstraintNamespace ("kubermatic") could be different from the namespace set for the Default Constraint Controller in the `seed-cluster-manager` . `kubermatic-api` also has the `namespace` option so maybe you can initialize the DefaultClusterProvider with it and use it through there somehow?

https://github.com/kubermatic/kubermatic/blob/master/cmd/kubermatic-api/options.go#L109

Comment (imharshita) [reply pkg/handler/v2/constraint/constraint.go]: done

**Pull request merged**